### PR TITLE
Code Review

### PR DESCRIPTION
**Pull request by edubew**
# Code review practice
- Perform a code review on a past project from microverse trials

### CODE REVIEW

**Review by edubew — COMMENTED** [review]
# Highlights
- Good job
- The workflow is good
- Add linters to your project

**Comment by sja-thedude** [comment]
> # Highlights
> * Good job
> * The workflow is good
> * Add linters to your project

- [ ] Kindly make sure to add a Required changes section after Highlights and write a comment like "Kindly make sure to add all the linters correctly by copying them from [here](https://github.com/microverseinc/linters-config/tree/master/html-css)"

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @edubew, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Good job so far in trying to review your code.

## Required 
- [x] Kindly check the quoted reply to rephrase your comment [here](https://github.com/edubew/code_review/pull/1#issuecomment-1108329943).
- [x] Kindly add Github inline comments at least once to leave review comments. Make sure to leave the comment under a line or block of code that fits the context the comment is about.
_I have attached a video which shows how you can add an inline comment._

https://user-images.githubusercontent.com/70269507/165065141-85b8ce37-ff27-4c19-ba41-ac6d6b28fd3d.mov

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    
+  }
+  li{
+      text-decoration: none;
+      display: inline;
+    
+  }
```
Comment (edubew) [inline style.css:22]: Kindly remove the extra space to make the code cleaner.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      }
+    }
+    a{
+        list-style: none;
+        text-decoration: none;
+    }
+    a.profile-link{
+      text-align: right;
+
+    }
```
Comment (edubew) [inline style.css:47]: Kindly include this part of the code inside the media query to make to make the elements responsive to different screen sizes

File: index.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        <section>
+            <div>
+                <h1 id="welcome-section">My Portfolio</h1>
+                <h2 id="project-title">This is the title</h2>
+                <div class="link" aria-colspan="2">
+                <a href="https://www.w3schools.com/" target="_blank">W3schools</a>
+                </div>
+                <a id="profile-link" href="https://github.com/microverseinc/fcc-test-suite-template">Github FCC</a>
+
+            </div>
```
Comment (edubew) [inline index.html:26]: Kindly remove the extra space here for a cleaner code

**Review by edubew — COMMENTED** [review]
# Required
Kindly check the comments below and make changes

**Review by gtekle — APPROVED** [review]
Hi @edubew :wave: , 

# STATUS: APPROVED 🎉🎉🎉

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Used polite suggestions
:heavy_check_mark: Added inline comments

## Optional suggestions
- [ Optional ] You should use appropriate options at the bottom when you do the review. If you requested required changes, you should as well select the Request changes option as in the screenshot below.
review-past-proj


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by edubew** [comment]
Hello @gtekle, Thank you for the review.
In your optional comments, you suggested that I should use the appropriate options button, for instance, "request changes" for this exercise but since I am the author of the pull request that I am reviewing, that option is blocked for me and I can only use the comments option

**Comment by gtekle** [comment]
Thanks, @edubew :+1: 
Noted!

**Pull request merged**